cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -59,6 +59,14 @@ OPTIONS
        --graalvm-build-options
            Additional build options to be passed to the GraalVM native image.
 
+       --runtime-management-included
+           Include the dependencies that are required to enable runtime package
+           management service.
+
+       --enable-service-publish
+          Include the dependencies that are required to enable runtime package
+          management service for WSO2 API manager.
```

Comment:
I would prefer to make this a generic feature, where WSO2 APIM is one of the service catalogs we support.